opengrok-web/src/main/webapp/list.jsp

```diff
@@ -362,6 +375,10 @@ Click <a href="<%= rawPath %>">download <%= basename %></a><%
             Util.dumpXref(out, xrefFile, compressed, request.getContextPath());
     %></pre>
 </div><%
+        } else {
+            String error = "Failed to get xref file";
+%>
+<p class="error"><%= error %></p><%
```

Comment:
you don't have to do the variable expansion, do you?

Reply:
what do you mean ?

Reply:
`<p class="error">Failed to get xref file</p>`

Reply:
I see, yes. Originally I wanted to make the error to include the path to the file however this is not necessary.